test/integration/configuration.test.js

```diff
@@ -17,35 +17,39 @@ if (dialect === 'sqlite') {
 
 describe(Support.getTestDialectTeaser('Configuration'), () => {
   describe('Connections problems should fail with a nice message', () => {
-    it('when we don\'t have the correct server details', async () => {
-      const options = {
-        logging: false,
-        host: 'localhost',
-        port: 19999, // Wrong port
-        dialect
-      };
-
-      const constructorArgs = [
-        config[dialect].database,
-        config[dialect].username,
-        config[dialect].password,
-        options
-      ];
-
-      let willBeRejectedWithArgs = [[Sequelize.HostNotReachableError, Sequelize.InvalidConnectionError]];
+    if (dialect != 'db2') {
+      it('when we don\'t have the correct server details', async () => {
+        const options = {
+          logging: false,
+          host: 'localhost',
+          port: 19999, // Wrong port
+          dialect
+        };
 
-      if (dialect === 'sqlite') {
-        options.storage = '/path/to/no/where/land';
-        options.dialectOptions = { mode: sqlite3.OPEN_READONLY };
-        // SQLite doesn't have a breakdown of error codes, so we are unable to discern between the different types of errors.
-        willBeRejectedWithArgs = [Sequelize.ConnectionError, 'SQLITE_CANTOPEN: unable to open database file'];
-      }
+        const constructorArgs = [
+          config[dialect].database,
+          config[dialect].username,
+          config[dialect].password,
+          options
+        ];
 
-      const seq = new Sequelize(...constructorArgs);
-      await expect(seq.query('select 1 as hello')).to.eventually.be.rejectedWith(...willBeRejectedWithArgs);
-    });
+        let willBeRejectedWithArgs = [[Sequelize.HostNotReachableError, Sequelize.InvalidConnectionError]];
+
+        if (dialect === 'sqlite') {
+          options.storage = '/path/to/no/where/land';
+          options.dialectOptions = { mode: sqlite3.OPEN_READONLY };
+          // SQLite doesn't have a breakdown of error codes, so we are unable to discern between the different types of errors.
+          willBeRejectedWithArgs = [Sequelize.ConnectionError, 'SQLITE_CANTOPEN: unable to open database file'];
+        }
+
+        const seq = new Sequelize(...constructorArgs);
+        await expect(seq.query('select 1 as hello')).to.eventually.be.rejectedWith(...willBeRejectedWithArgs);
+      });
+    }
 
     it('when we don\'t have the correct login information', async () => {
+      const willBeRejectedWithArgs = [[Sequelize.HostNotReachableError, Sequelize.InvalidConnectionError]];
```

Comment:
Why is this added? Because it's not here for db2? Maybe in that case add it somewhere for the entire file before it runs the first test?

Reply:
Yes, this is needed for db2 and I followed the same format as the previous test case i.e. "when we don't have the correct server details". Am I right?